.github/workflows/validate.yml

```
@@ -0,0 +1,107 @@
+name: Validate Assets
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)